.eslintrc.cjs

```diff
@@ -65,7 +65,7 @@ module.exports = {
         'import/prefer-default-export': 'off',
 
         'arrow-parens': ['warn', 'as-needed'],
-        'max-len': ['warn', {code: 150}],
+        'max-len': ['warn', {code: 160}],
```

Comment:
length of link to docs in was bigger than 150, so increased to 160

Reply:
ok